l10n_it_partially_deductible_vat/__openerp__.py

```diff
@@ -24,27 +24,25 @@
 ##############################################################################
 
 {
-    "name" : "Italy - Partially Deductible VAT",
-    "version" : "0.1",
-    "depends" : ['account'],
-    "author" : "OpenERP Italian Community,Odoo Community Association (OCA)",
+    "name": "Italy - Partially Deductible VAT",
+    "version": "0.1",
+    "depends": ['account'],
+    "author": "OpenERP Italian Community,Odoo Community Association (OCA),"
+              "SimplERP srl",
     "description": """
```

Comment:
Ah, ok
I have to put another test everyway.
Il 31/lug/2015 16:31, "Lorenzo Battistini" <notifications@github.com> ha
scritto:

> In l10n_it_partially_deductible_vat/__openerp__.py
> <https://github.com/OCA/l10n-italy/pull/127#discussion_r35978640>:
>
> > @@ -24,27 +24,25 @@
> >  ##############################################################################
> >
> >  {
> > -    "name" : "Italy - Partially Deductible VAT",
> > -    "version" : "0.1",
> > -    "depends" : ['account'],
> > -    "author" : "OpenERP Italian Community,Odoo Community Association (OCA)",
> > +    "name": "Italy - Partially Deductible VAT",
> > +    "version": "0.1",
> > +    "depends": ['account'],
> > +    "author": "OpenERP Italian Community,Odoo Community Association (OCA),"
> > +              "SimplERP srl",
> >      "description": """
>
> @sergiocorato <https://github.com/sergiocorato> actually I was talking
> about the description item of __openerp__.py dictionary
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OCA/l10n-italy/pull/127/files#r35978640>.
>